src/js/discharge-wizard/components/InstructionsPage.jsx

```diff
@@ -110,8 +119,8 @@ class InstructionsPage extends React.Component {
                             </div>
                           </li>
                           <li itemScope itemType="http://schema.org/Question">
-                            <button className="usa-button-unstyled usa-accordion-button" aria-controls="dbq5" itemProp="name">What if I have a DD-215 showing an upgraded discharge, but my DD-214 is still incorrect?</button>
-                            <div id="dbq5" className="usa-accordion-content" itemProp="acceptedAnswer" itemScope itemType="http://schema.org/Answer">
+                            <button className="usa-button-unstyled usa-accordion-button" aria-controls="dbq5" itemProp="name" aria-expanded={this.state.activeFAQ === 'q5'} onClick={this.handleFAQToggle} name="q5">What if I have a DD-215 showing an upgraded discharge, but my DD-214 is still incorrect?</button>
```

Comment:
Can only one accordion be expanded at a time? If we should be able to expand multiple, perhaps we should keep track of each question in state.

Reply:
That makes sense. I’ll make this change. Ditched the USWDS script because it doesn’t make sense for this use case (and is not designed to reinitialize inside of React components )